### DISCUSSION TITLE
Get Link to Page in Markdown

### DISCUSSION THREAD

**Discussion by JuStTheDev** [hexojs/hexo#5490] [Question / Answer]
Hi there,
I wonder if there is a possibility to **get a link to a specific page** in markdown. My use case is, that I am writing a post (.md) and want to link to a page. I wonder if the only possibility is to have the path explicitly as in `href="/[myprefix]/path/to/page` (my _config.yml has a `url: https://domain/[myprefix]` set). The problem with this is when I change the `myprefix`, I have to correct hard-coded paths everywhere. Relative paths are also not perfect because the relation depends on the `permalink`-value in `_config.yml`. 

I know there is a solution for linking to posts [here](https://hexo.io/docs/tag-plugins#Include-Posts) but I was not able to use this for linking to a page.

I would be fine with an answer "not possible" but I would also be interested in your practices in linking to internal pages.

**Comment by uiolee** [comment]
https://hexo.io/docs/tag-plugins#URL
